composables/data/articlePages.ts

```diff
@@ -108,4 +108,51 @@ export function normalizeArticlePage(article: Record<string, any>): ArticlePage
 // Transform a list of article page data from the API into a simpler and typed format
 export function normalizeFindArticlePagesResponse(articlesResponse: any): ArticlePage[] {
     return articlesResponse.value.items?.map(normalizeArticlePage)
+}
+
+// Transform the featured article page data from the API into a simpler and typed format
+// The home page featured article should display only the first story in the content collection
+export function normalizeHomePageFeaturedArticle(article: Record<string, any>): ArticlePage {
```

Comment:
I don't think we need to duplicate this whole function just to call it on a deeper nested value, we can just call `normalizeArticlePage(article.value.pageCollectionRelationship?.[0].pages?.[0])`
Unless I'm misunderstanding what this is doing.

Reply:
Another good idea :)